src/test/java/io/mosip/mimoto/util/CredentialIssuerWellknownResponseValidatorTest.java

```diff
@@ -227,7 +229,48 @@ public void shouldDetectMissingMandatoryFieldsOfCredentialDefinitionResponseDtoR
         InvalidWellknownResponseException invalidWellknownResponseException = assertThrows(InvalidWellknownResponseException.class, () ->
                 credentialIssuerWellknownResponseValidator.validate(response, validator)
         );
+
         assertTrue(invalidWellknownResponseException.getMessage().contains("type: must not be empty"));
         assertTrue(invalidWellknownResponseException.getMessage().contains("credentialSubject: must not be empty"));
     }
+
+    @Test
+    void shouldThrowInvalidWellKnownResponseExceptionWhenMandatoryFieldsAreNotPresentInMsoMdocVc() {
+        CredentialsSupportedResponse credentialSupportedResponse = getCredentialSupportedResponse("CredentialType1", "mso_mdoc");
```

Comment:
Lets Follow the AAA pattern the test case 

Move all assertions together or we can create a new test case for that if we want the segment to be together.